cmd/ocm-backplane/managedJob/getManagedJob.go

```diff
@@ -91,6 +94,11 @@ func newGetManagedJobCmd() *cobra.Command {
 					return err
 				}
 
+				err = deprecation.CheckResponseDeprecation(resp)
+				if errors.Is(err, deprecation.ErrDeprecation) {
+					logger.Warnf("The server indicated that this version of backplane-cli is deprecated, please update as soon as possible.")
```

Comment:
It seems this API deprecates error message duplicates in several places in this PR. Can the error be handled in a single place? 
May be utilize this func : https://github.com/openshift/backplane-cli/blob/594bdf05cc83373296422dda5275974cf5c82121/pkg/utils/util.go#L137